sfge/src/main/java/com/salesforce/graph/symbols/apex/schema/FieldSet.java

```diff
@@ -141,4 +142,19 @@ private Optional<ApexValue<?>> _applyMethod(
         }
         return Optional.empty();
     }
+
+    @Override
```

Comment:
Missing equals/hashcode were causing these items to get lost in HashSet. I've added it to all the Standard library schema types that were missing this.